src/libraries/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -90,6 +99,7 @@ public async Task WriteFrameAsync(Frame frame)
 
         // Read until the buffer is full
         // Return false on EOF, throw on partial read
+#if !NETFRAMEWORK
         private async Task<bool> FillBufferAsync(Memory<byte> buffer, CancellationToken cancellationToken = default(CancellationToken))
         {
             int readBytes = await _connectionStream.ReadAsync(buffer, cancellationToken).ConfigureAwait(false);
```

Comment:
Can we give `Stream.ReadAsync` the same extension method treatment as `WriteAsync` to collapse this `#if`?